github/util.py

```diff
@@ -221,3 +222,82 @@ def retrieve_email(username: str):
         )
     )
 
+def _create_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str
+):
+    # passed GitHub object must have org. admin authorization to create a team
+    organization = github.organization(organization_name)
+    team_list = list(filter(lambda t: t.name == team_name, organization.teams()))
```

Comment:
is we are only interested in the first (and only) match, it might be more elegant to iterate over all teams like so:

for team in organization.teams():
 if team.name == team_name:
   return util.verbose(....)

but in the end this is more of a stylistic question